pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -18,63 +17,137 @@ import (
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/ownerutil"
 )
 
+const (
+	operatorGroupAnnotationKey          = "olm.operatorGroup"
```

Comment:
One of us is going to have to rebase a bit. I ended up pulling this into types so the e2e can also use them in https://github.com/operator-framework/operator-lifecycle-manager/pull/616/. Will finish reviewing this tomorrow.

Reply:
I think #616 will need to change (slightly) anyway - see the `operatorGroupForActiveCSV ` method.